build.sbt

```diff
@@ -0,0 +1,47 @@
+import com.typesafe.sbt.SbtGit._
+
+name := "lenthall"
+
+version := "0.13-SNAPSHOT"
+
+organization := "org.broadinstitute"
+
+scalaVersion := "2.11.7"
+
+libraryDependencies ++= Seq(
+  "com.typesafe" % "config" % "1.2.1",
```

Comment:
Would another project using lenthall (e.g. Tyburn) need to include these in their own SBT now or is this considered part of the common code?